atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/monitoring/TimestampTrackerTest.java

```diff
@@ -189,6 +189,14 @@ public Long get() {
         }
     }
 
+    private TimestampTrackerImpl createTrackerWithClock(Clock clock) {
+        return new TimestampTrackerImpl(clock, timelockService, cleaner);
+    }
+
+    private TimestampTrackerImpl createTracker() {
```

Comment:
nit^2: I'd prefer `createDefaultTracker` (mainly because seeing `createTrackerWithClock(Clock.defaultClock())` and this looks kind of weird)